### DISCUSSION TITLE
IPL/IDE export support

### DISCUSSION THREAD

**Discussion by ghost** [Parik27/DragonFF#305] [Ideas]
Good day everyone, the function of importing a map (ipl /ide) has long been implemented by one of the contributors, but the community lacks support for exporting the map

I hope someone can add this to the addon :)

**Comment by MadGamerHD** [comment]
we'll i hope my project helps

**Reply by ghost** [reply]
> we'll i hope my project helps

It would be great, or better yet, more logical to integrate your addon with IDE/IPL export into this large tool

**Reply by MadGamerHD** [reply]
our current priority is making sure GTASceneSync works smoothly as a dedicated solution
